benchopt/plotting/generate_html.py

```diff
@@ -203,8 +202,49 @@ def get_results(fnames, kinds, root_html, benchmark_name, copy=False):
             f"{result['fname_short'].replace('.csv', '.html')}"
         )
 
+        result['json'] = benchmark_dataframe_to_json(df)
+
     return results
 
+def benchmark_dataframe_to_json(df):
+    json_data = {}
+
+    # Retrieve interesting data from the dataframe
+    datasets = df['data_name'].unique() # Array of str
+    objectives = df['objective_name'].unique() # Array of str
+    solvers = df['solver_name'].unique() # Array of str
+
+    columns = ['objective_value', 'objective_support_size', 'objective_duality_gap']
```

Comment:
The name of the objective columns can change from one benchmark to another.
You should to this for all columns in `df` starting with `objective_`